back/app/services/user_service.rb

```diff
@@ -27,8 +27,6 @@ def update_in_admin_api(user, user_params, confirm_user)
       user.update(user_params)
     end
 
-    # If SSO method doesn't provide email, user enters it manually
-    # (`update_in_web_api` is called)
```

Comment:
Removed this comment because it's outdated (not due to changes in this PR but because of the feature in general)